mailpoet/assets/js/src/form_editor/components/sidebar/sidebar.tsx

```diff
@@ -1,9 +1,21 @@
 import { useEffect } from 'react';
 import { useDispatch, useSelect } from '@wordpress/data';
+import {
+  ReduxStoreConfig,
+  StoreDescriptor,
+} from '@wordpress/data/build-types/types';
 import { DefaultSidebar } from './default_sidebar';
 import { PlacementSettingsSidebar } from './placement_settings_sidebar';
 import { store } from '../../store';
 
+// workaround for block editor store useSelect
```

Comment:
Was this later removed (moved to `types/...` as per https://github.com/mailpoet/mailpoet/pull/4786/commits/5638e314cd633498efe6a247dcc6328775d77175)? Need to check.